clang/lib/CIR/CodeGen/CIRGenFunction.cpp

```diff
@@ -993,7 +993,8 @@ void CIRGenFunction::StartFunction(GlobalDecl GD, QualType RetTy,
     llvm_unreachable("NYI");
 
   if (FD && getLangOpts().OpenCL) {
-    llvm_unreachable("NYI");
+    // TODO(cir): Emit OpenCL kernel metadata
+    assert(!MissingFeatures::openCL());
```

Comment:
Not sure about this, if you are not implementing this as part of this PR, why are you letting it pass?

Reply:
Converting it to MissingFeatures is required for test cases in this PR.